### PR TITLE
Tia duplication bugfix

### PR DESCRIPTION
**Pull request by EvgeniyMeteliza**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fixed problem with incidents duplication.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA/GroupIB_TIA_description.md [hunk 1]
```diff
@@ -10,4 +10,8 @@
 3. Your server URL is the same as your TI&A web interface URL.  
 4. Your username is the email that you use to enter in the web interface.
 5. Set classifier and mapper with Group-IB Threat Intelligence & Attribution classifier and mapper or with our own if you want so.
-6. Don't forget to contact Group-IB to whitelist your Cortex IP or public IP of a proxy that you are using with Cortex.
\ No newline at end of file
+6. Go to the Settings->Integrations->Pre-Processing Rules and set up the pre-processing rule:
```
Comment (Noy-Maimon) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA/GroupIB_TIA_description.md:13]:
```suggestion
6. Go to Settings->Integrations->Pre-Processing Rules and set up the pre-processing rule:
```

File: Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from CommonServerPython import *  # noqa: F401
+result = True
+inc = demisto.incident()
+custom_fields = inc.get("CustomFields", {})
+del inc["CustomFields"]
+del inc["labels"]
+del inc["occurred"]
+del inc["sla"]
+inc.update(custom_fields)
+demisto.error(custom_fields)
```
Comment (Noy-Maimon) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py:11]: why is this as `demisto.error`? what is the purpose? if this is not an actual error (as it looks like) perhaps there is a better command for that purpose.

File: Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+gibid = custom_fields.get('gibid')
+demisto.error(gibid)
```
Comment (Noy-Maimon) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py:13]: same here

**Review by Noy-Maimon — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please add the requested changes for the comments i left on some of the files.
Plus, please add a unit test for the new script. 
Let me know if i can help in anyway.

File: Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+incid = demisto.executeCommand("getIncidents", {"query": "gibid: {0} and -status:Closed".format(gibid)})
+total = int(incid[0]["Contents"]["total"])
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py]: done

File: Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+if total > 0:
+    result = False
+    incident_id = incid[0]["Contents"]["data"][total - 1]["id"]
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py]: done

File: Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_0_2.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### Group-IB Threat Intelligence & Attribution
+- Added instructions for the creation of the pre-processing rule to the description.
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_0_2.md]: done

File: Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_0_2.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+
+#### Scripts
+##### New: GIBIncidentUpdate
+- This script prevents duplication of existing incidents. (Available from Cortex XSOAR 6.0.0).
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_0_2.md]: done

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA/GroupIB_TIA_description.md (hunk 1 above)
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA/GroupIB_TIA_description.md]: done

File: Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py (hunk 2 above)
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py]: done

File: Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py (hunk 3 above)
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py]: done

**Comment by EvgeniyMeteliza** [comment]
Sorry for the bad pull request.

**Review by Noy-Maimon — APPROVED** [review]

**Pull request merged**